OutOfSchool/OutOfSchool.DataAccess/Models/ProviderSectionItem.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace OutOfSchool.Services.Models
+{
+    public class ProviderSectionItem : IKeyedEntity<Guid>
+    {
+        public Guid Id { get; set; }
```

Comment:
as these items won't be directly exposed via a controller, maybe we can use Long as ID? save a few bytes per record. Not critical comparing to the size of other fields, but just a thought :)